src/tokens.md

```diff
@@ -486,10 +486,15 @@ The two values of the boolean type are written `true` and `false`.
 ## Lifetimes and loop labels
 
 > **<sup>Lexer</sup>**  
+> LIFETIME_TOKEN
+> &nbsp;&nbsp; &nbsp;&nbsp; `'` [IDENTIFIER_OR_KEYWORD][identifier]  
+> &nbsp;&nbsp; | `'_`  
+>  
 > LIFETIME_OR_LABEL:  
 > &nbsp;&nbsp; &nbsp;&nbsp; `'` [IDENTIFIER][identifier]
 
-Lifetime parameters and [loop labels] both use this syntax.
+Lifetime parameters and [loop labels] use LIFETIME_OR_LABEL tokens. Slightly
+more tokens are accepted by the lexer, and can be used in macros, for example.
```

Comment:
I think it would read slightly better if the "for example" was after the "and". I also think the "and" would read better as a "which". It'd be nice to state or list exactly which other tokens are accepted by the lexer and where else other than macros they can be used.